extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -61,7 +61,7 @@ export default function initWADOImageLoader(
       // For now we use image/jls and image/x-jls because some servers still use the old type
       // http://dicom.nema.org/medical/dicom/current/output/html/part18.html
       const xhrRequestHeaders = {
-        Accept: 'multipart/related; type="application/octet-stream"',
+        Accept: (appConfig.ommitQuotationFromMultipartRequest) ? 'multipart/related; type=application/octet-stream' : 'multipart/related; type="application/octet-stream"'
```

Comment:
I'd rather have this just be a straight string configured in the appConfig, so that the entire string can be configured if needed.

Reply:
@wayfarer3130,  I envision using this configuration to manage other multi-part requests rather than limiting it to this specific use-case, so it behaves as a global configuration. 
For example, when the dcmjs/webclient library enable this configuration, as an end user, you will turn this off/on from this one location and we internally configure the library and any other location in OHIF that uses this quotation accordingly.